<?xml version="1.0"?>

<launch>
    <!-- static_transform_publisher x y z yaw pitch roll frame_id child_frame_id -->
    <node pkg="tf2_ros" type="static_transform_publisher" name="static_world_publisher" args="0.0 0.0 0.0 0.0 0.0 0.0 world base_link" />

    <rosparam param="/vr_node/publish_joy">True</rosparam>
    <rosparam param="/vr_node/publish_twist">True</rosparam>
    <rosparam param="/vr_node/x_offset">0.0</rosparam>
    <rosparam param="/vr_node/y_offset">0.0</rosparam>
    <rosparam param="/vr_node/z_offset">2.0</rosparam>
    <rosparam param="/vr_node/yaw_offset">0.0</rosparam>
    <rosparam param="/vr_node/pitch_offset">1.57079632679</rosparam>
    <rosparam param="/vr_node/roll_offset">0.0</rosparam>

    <!-- Launch node through steam with shell script -->
    <node name="vive_node" pkg="vive_bridge" type="launch.sh" output="screen" required="true"/>
</launch>